smoke-test/config.yaml

```diff
@@ -0,0 +1,33 @@
+General:
+  ConfigurationVersion: 2
+  MinRefineryVersion: v2.0
+  ConfigReloadInterval: 50s
```

Comment:
Just curious on why you wanted to set this to be 50s?

Reply:
These configs were copied from the ones currently in the `test/` directory and then I added the TLS settings to them. I have no informed opinion about their contents other than enabling TLS.

Reply:
I removed a few things from this config to have them default to … the defaults.